ethereum/evmtool/src/main/java/org/hyperledger/besu/evmtool/benchmarks/BenchmarkExecutor.java

```diff
@@ -205,12 +205,21 @@ protected double runPrecompileBenchmark(
             });
 
     int executions = 0;
-    long elapsed = 0;
-    startNanoTime = System.nanoTime();
-    while (executions < execIterations && elapsed < execTimeInNano) {
-      contract.computePrecompile(arg, fakeFrame);
+    long totalElapsed = 0;
+    boolean isInvalidCase = false;
+
+    while (executions < execIterations && totalElapsed < execTimeInNano) {
+      long iterationStart = System.nanoTime();
+      final var result = contract.computePrecompile(arg, fakeFrame);
+      long iterationElapsed = System.nanoTime() - iterationStart;
+
+      if (result.output() == null) {
+        isInvalidCase = true;
```

Comment:
I don't think this case is possible (with main branch) due to the this earlier in the method:
```
    if (contract.computePrecompile(arg, fakeFrame).output() == null) {
      throw new RuntimeException("Input is Invalid");
    }
```